### PR TITLE
fixed the wrong keyboard shortcuts for vscode

### PR DESCRIPTION
**Pull request by aayushbisen**
Fixed the wrong keyboard shortcuts for **Hot Reload** and **Hot Restart** on [Visual Studio Code](https://flutter.dev/docs/development/tools/vs-code) page.

### CODE REVIEW

File: src/docs/development/tools/vs-code.md [hunk 1]
```diff
@@ -238,15 +238,15 @@ You can also define custom snippets by executing
 
 ### Keyboard shortcuts
 
-**Hot reload**
-: During a debug session, clicking the **Restart** button on the
-  **Debug Toolbar**, or pressing `Ctrl`+`Shift`+`F5`
-  (`Cmd`+`Shift`+`F5` on macOS) performs a hot reload.
+**Hot Reload**
```
Comment (sfshaza2) [inline src/docs/development/tools/vs-code.md:241]: @aayushbisen, we use sentence case for titles and headings:  https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

Please undo the changes to the headings. thx

Comment (aayushbisen) [reply src/docs/development/tools/vs-code.md]: Done.
